SphericalShell/README.md

```diff
@@ -0,0 +1,17 @@
+# Spherical Shell Problem ReadMe File
+
+Describes the basic parameters of the spherical shell/point source problem.
+
+## 1. Source Type
+* Point source of monoenergetic (14 MeV) neutrons at origin (0,0,0)
+## 2. Materials
+* Tungsten shell
```

Comment:
Looking at your problem definition in OpenMC, it would be good to add the isotopic composition and density of this material to this description 

```suggestion
* Tungsten shell - natural abundances, density = 19.6 g/cm3
```